WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionSummaryViewModel.cs

```diff
@@ -56,7 +57,7 @@ public void UpdateTransaction(BuildTransactionResult transactionResult, Transact
 		var destinationAmount = _transaction.CalculateDestinationAmount();
 		AmountText = $"{destinationAmount.ToFormattedString()} BTC";
 
-		var fee = _transaction.Fee;
+		Money fee = _transaction.Fee;
```

Comment:
Why was this needed?

Reply:
For nullability, changing it back for consistency.